geode-core/src/distributedTest/java/org/apache/geode/distributed/internal/P2PMessagingConcurrencyDUnitTest.java

```diff
@@ -283,8 +383,11 @@ public void toData(final DataOutput out, final SerializationContext context)
       out.writeInt(length);
 
       final byte[] payload = new byte[length];
-      random.nextBytes(payload);
-
+      if (RANDOMIZE_PAYLOAD_CONTENT) {
```

Comment:
Make a separate test or parameter to this test.

Reply:
reverted changes to this test